core/pta/bcm/bnxt.c

```diff
@@ -17,6 +17,9 @@
 
 enum pta_bnxt_cmd {
 	PTA_BNXT_FASTBOOT = 0,
+	PTA_BNXT_HEALTH_STATUS,
+	PTA_BNXT_HANDSHAKE_STATUS,
+	PTA_BNXT_CRASH_DUMP_COPY,
```

Comment:
Minor: would be good to document the parameters for these new commands, it's good practice, but not strictly required either.

Reply:
Done